noma/lnd.py

```diff
@@ -54,7 +56,8 @@ def encodemacaroons(macaroonfile=cfg.MACAROON_PATH, tlsfile=cfg.TLS_CERT_PATH):
 
 
 def connectstring(hostname=cfg.URL_GRPC, macaroonfile=cfg.MACAROON_PATH, tlsfile=cfg.TLS_CERT_PATH):
-    result = encodemacaroons(macaroonfile=macaroonfile, tlsfile=tlsfile)
+    """Show lndconnect string for remote wallets such as Zap"""
```

Comment:
Appreciate comments.

Reply:
I've simplified this non-sense now. I believe `macaroonfile` and `tlsfile` were being explicitly cast to string from Path objects before and this was left dangling